medicaltorch/datasets.py

```diff
@@ -107,7 +107,7 @@ def __init__(self, input_filename, gt_filename, metadata=None, cache=True, canon
 
         if self.metadata:
```

Comment:
We should remove all this block since we never need input_filename or gt_filename from the metadata dictionary

Reply:
[input_metadata][gt_filename] is used in this PR https://github.com/neuropoly/ivado-medical-imaging/pull/114/files#diff-b2f49d8e8c234ff61baaf83517f958edR682 , so finally it might be better to keep it

Reply:
But @charleygros only needs the filename name of the corresponding file, not a list of all filenames right?

Reply:
I think so yep!

Reply:
indeed